File: wave.java
import java.util.*;
import java.io.*;

public class wave {
  public static void main(final String[] args) {
    final Scanner in = new Scanner(System.in);
    int n = in.nextInt();

    while (n-- > 0) {
      final int a = in.nextInt(), b = in.nextInt();

      String ans = "one";

      if (a < b) {
        ans = "two";
      }

      System.out.println(ans);
    }

    in.close();
  }
}

File: wave.cpp
#include <bits/stdc++.h>

int main(void) {
  int n;
  std::cin >> n;
  while (n--) {
    int a, b;
    std::cin >> a >> b;

    std::string ans = "one";
    if (a < b) {
      ans = "two";
    }

    std::cout << ans << '\n';
  }

  return 0;
}
